apps/dashboard/test/system/batch_connect_sessions_test.rb

```diff
@@ -40,7 +40,8 @@ def create_test_file(dir, token: 'sys/bc_paraview', title: 'Paraview')
   def stub_scheduler(state, cores: 1, nodes: 1)
     info = OodCore::Job::Info.new(
       id: test_job_id, status: state.to_sym, procs: cores.to_i, 
-      allocated_nodes: Array.new(nodes.to_i, { name: 'foo', features: [] })
+      allocated_nodes: Array.new(nodes.to_i, { name: 'foo', features: [] }),
```

Comment:
I think i can yank this, it was part of my setup before i used `ClimateControl` to set the config bit for this specific test and not break the others.

Reply:
Oh, actually this is very much needed to have a result to process :face_palm